src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss

```diff
@@ -1,7 +1,13 @@
 @import 'src/styles/common';
 @import 'constants';
 
-.defaultTranscriptsList {
+.header {
+  border-bottom: 1px solid $orange;
+  line-height: 2;
+  width: 95%;
```

Comment:
Why is the header narrower than the whole list? Couldn’t find in the XD what this might be referring to.

Reply:
Removed this